openpype/vendor/python/common/capture.py

```diff
@@ -746,6 +758,14 @@ def _applied_viewport_options(options, panel):
         cmds.modelEditor(panel, edit=True, **options)
     except TypeError as e:
         logger.error("Cannot apply options {}".format(e))
```

Comment:
Should this error logging be removed since we are logging the "Failing to apply" message?

Atm we get 2 error messages when failing to apply.

Reply:
I agree. Thanks.
```suggestion
```